.env.development

```
@@ -104,3 +110,4 @@ VITE_FEATURE_STONFI_SWAP=true
 VITE_FEATURE_YIELDS_PAGE=true
 VITE_FEATURE_EARN_TAB=true
 VITE_FEATURE_YIELD_MULTI_ACCOUNT=true
+VITE_FEATURE_GENERIC_EVM_CHAINS=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FEATURE_GENERIC_EVM_CHAINS key should go before the VITE_FEATURE_KATANA key

(UnorderedKey)